helm_deploy/values-preprod.yaml

```diff
@@ -38,19 +38,14 @@ generic-service:
     FEATURE_EVENT_CSIP_REPORTS-DELETED: false
```

Comment:
Code relating to the actual child table deletes will be removed in a future PR, once all delete code (for top level CSIP) is complete